packages/hint/src/lib/types.ts

```diff
@@ -73,6 +73,15 @@ export type UserConfig = {
     ignoredUrls?: IgnoredUrl[];
 };
 
+export type ConfigFiles = {
+    '.hintrc'? : string | string[];
+    '.hintrc.js'?: string | string[];
+    'hintrc.json'?: string | string[];
+    'package.json'?: string | string[];
+    'hintConfig'?: string | string[];
+    [k: string]: string | string[] | undefined;
+};
```

Comment:
I would reuse the ConfigFile type so we can avoid duplicating the config names. What do you think?
```typescript
export type ConfigFiles = {
    [key in ConfigFile]?: string | string[]
};
```